vite.config.js

```diff
@@ -48,5 +48,10 @@ export default defineConfig({
     environment: "jsdom",
     globals: true,
     setupFiles: ["./tests/setup.js"],
+    server: {
+      deps: {
+        inline: ["@kbss-cvut/s-forms"],
```

Comment:
Can we describe why we need this?

Reply:
@shellyear I created https://github.com/kbss-cvut/s-forms/issues/296 but cannot assign you to the ticket yet -- please confirm pending invitation there (should have it in email).

Reply:
@blcham accepted the invitation